requirements.txt

```diff
@@ -0,0 +1,4 @@
+Django>=2.1.3,<2.2.0
+djangorestframework>=3.9.0,<3.10.0
```

Comment:
Also I recommend pinning the exact versions of the packages to avoid problems. Some packages may introduce breaking changes due to inconsistent versioning numbers.

Reply:
Versions upgraded to specific ones